tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -540,6 +541,7 @@ def test_scheduler_create_lsf_driver():
     assert str(driver._bhist_cmd) == bhist_cmd
     assert driver._queue_name == queue_name
     assert driver._resource_requirement == lsf_resource
+    assert driver._exclude_hosts == exclude_host.split(",")
```

Comment:
hmm, testing implementation details.. If we refactor the driver this test will be irritating, we don't want to assert on how it is implemented, only that it is doing the right thing.